woo-razorpay.php

```diff
@@ -1076,6 +1124,8 @@ public function updateOrder(& $order, $success, $errorMessage, $razorpayPaymentI
                 $order->payment_complete($razorpayPaymentId);
                 $order->add_order_note("Razorpay payment successful <br/>Razorpay Id: $razorpayPaymentId");
 
+                $this->transferFromPayment($orderId, $razorpayPaymentId); // creates transfers from payment
```

Comment:
Can we pass order ref to this function instead of passing order id and fetchin the order details again?

Reply:
Passing the $orderId as we need this to get the order items.